### ISSUE TITLE
JSON for device name?

### ISSUE THREAD

**Issue by PcTim** [powtac/Autarc_LAN_User_Stats#50]
Maybe we should change the communication for the Device-Input Form also to JSON?
The disadvantage is, that we need to use ajax. The advantage, that we can handle this request like the new device request.
